temperature/utils.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+import httpx
+from fastapi import Depends, HTTPException
+from sqlalchemy.ext.asyncio import AsyncSession
+from dotenv import load_dotenv
+
+from weather.crud import get_all_cities
+from dependencies import get_db
+
+load_dotenv()
+
+API_URL = os.getenv("API_URL")
```

Comment:
Ensure that the `API_URL` environment variable is correctly set in your `.env` file. This variable is crucial for making requests to the external API.